internal/integrations/envsec/envsec.go

```diff
@@ -20,10 +32,23 @@ func Env(projectDir string) (map[string]string, error) {
 		return nil, err
 	}
 
-	return envsecList(projectDir)
+	var err error
+	envCache, err = envsecList(projectDir)
+
+	return envCache, err
 }
 
 func ensureEnvsecInstalled() error {
+	// In newer runx version this will return the paths
+	if err := runx.Install("jetpack-io/envsec"); err != nil {
+		return errors.Wrap(err, "failed to install envsec")
+	}
+
+	// temporary, mac only
+	home, _ := os.UserHomeDir()
+	path := filepath.Join(home, "Library/Caches/jetpack.io/pkgs/jetpack-io/envsec/v0.0.1/darwin/arm64/")
+	os.Setenv("PATH", path+string(os.PathListSeparator)+os.Getenv("PATH"))
```

Comment:
Why do we need this? Does runx not handle macOS?

Reply:
This is for testing. The runx library will now return the path (for all os's)